sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_inventory_v1/query.py

```diff
@@ -69,6 +69,24 @@ def create_last_modified_tmp_table(date, project, tmp_table_name):
 def create_query(date, source_project, tmp_table_name):
     """Create query for a source project."""
     return f"""
+        WITH labels AS (
+            SELECT table_catalog,
+                 table_schema,
+                 table_name,
+                 ARRAY_AGG(value) AS owners
+            FROM
+                (SELECT table_catalog, table_schema, table_name,
+                  ARRAY(
+                    SELECT as STRUCT ARR[OFFSET(0)] key, ARR[OFFSET(1)] value
+                    FROM UNNEST(REGEXP_EXTRACT_ALL(option_value, r'STRUCT\(("[^"]+", "[^"]+")\)')) kv,
+                    UNNEST([STRUCT(SPLIT(REPLACE(kv, '"', ''), ', ') as arr)])
```

Comment:
This parsing made me curious so I took a look at the `TABLE_OPTIONS` view and a string array of structs is an interesting way to store these 😅 I wonder why this was the choice...